rakelib/lint.rake

```diff
@@ -1,64 +1,31 @@
 # Used constants:
-# - WORKSPACE
+# none
 
 namespace :lint do
-  SWIFTLINT_VERSION = '0.25.0'.freeze
+  SWIFTLINT = 'Pods/SwiftLint/swiftlint'
 
-  desc 'Install swiftlint'
-  task :install do |task|
-    next if check_version
-
-    unless system('tty >/dev/null')
-      puts "warning: Unable to install SwiftLint #{SWIFTLINT_VERSION} without a terminal." \
-        "Please run 'bundle exec rake lint:install' from a terminal."
-      next
-    end
-
-    url = "https://github.com/realm/SwiftLint/releases/download/#{SWIFTLINT_VERSION}/SwiftLint.pkg"
-    tmppath = '/tmp/SwiftLint.pkg'
-
-    Utils.run([
-                "curl -Lo #{tmppath} #{url}",
-                "sudo installer -pkg #{tmppath} -target /"
-              ], task)
-  end
-
-  if File.directory?('Sources')
-    desc 'Lint the code'
-    task :code => :install do |task|
-      Utils.print_header 'Linting the code'
-      config = Pathname.getwd + '.swiftlint.yml'
-      config = config.exist? ? "--config #{config}" : ''
-      Utils.run(%(swiftlint lint --no-cache --strict --path Sources #{config}), task)
-    end
+  desc 'Lint the code'
+  task :code do |task|
+    Utils.print_header 'Linting the code'
+    config = Pathname.getwd + '.swiftlint.yml'
+    Utils.run(%(#{SWIFTLINT} lint --strict --path Sources --config "#{config}"), task)
```

Comment:
Same question, worth wondering if we should really keep those tasks in the Rakefile, especially since, now that they're integrated as Build Phases, I fear that we could miss keeping them in sync if we change those one day (like change the paths in one place but not in the other)…?